core/src/main/scala/hydra/core/connect/Connector.scala

```diff
@@ -60,4 +60,5 @@ case class RequestReceived(request: HydraRequest) extends HydraMessage
 case class RequestConfirmed(deliveryId: Long) extends HydraMessage
 
 case class HydraConnectIngestError(connectorId: String, source: String,
-                                   statusCode: Int, msg: String) extends HydraMessage
\ No newline at end of file
+                                   statusCode: Int, msg: String,
+                                   metadata: Map[String, String] = Map()) extends HydraMessage
```

Comment:
Let's use Map.Empty here instead of Map

Reply:
This doesn't belong here... I say we add this as a param in HydraNotifications, not HydraIngest